includes/class-sensei-course-progress.php

```diff
@@ -116,7 +116,7 @@ public function include_widgets() {
 	public function enqueue_styles () {
 		global $woothemes_sensei;
 
-		wp_register_style( $this->_token . '-frontend', esc_url( $this->assets_url ) . 'css/frontend.css', array(), $this->_version );
+		wp_register_style( $this->_token . '-frontend', esc_url( $this->assets_url ) . 'css/frontend.min.css', array(), $this->_version );
```

Comment:
After this change, and after removing the built `frontend.css` file, the CSS no longer loads when using the dev version of the plugin. Works fine in the built version.

Reply:
This was intentional. It's probably not a good idea for us to be developing and testing locally using the raw source code. Instead, it should be run through the build so that that part gets tested as well, and so that we're testing the code that is actually going to be deployed. In the past, I've dealt with this by creating symlinks from the source code directory to the plugin directory so that only the built files get copied over.

I debated creating a separate Gulp task for development mode that wouldn't minify / compress the CSS and JS, but because this repo doesn't have any JS, and I didn't think it was that useful for debugging CSS, I didn't bother. But I can do so if you'd like.

Reply:
What we need is a proper Readme with those instructions in there. Any objection to my doing that in a separate PR?

Reply:
Sounds good to me!